main/openvswitch/debian.readme.patch

```diff
@@ -0,0 +1,119 @@
+--- openvswitch-2.9.0/debian/openvswitch-switch.README.Debian
++++ openvswitch-2.9.0/debian/openvswitch-switch.README.Debian.new
```

Comment:
It’s quite weird to have `README.Alpine` in file `README.Debian`…

Reply:
```
        install -m644 debian/openvswitch-switch.README.Debian \
                "$pkgdir"/usr/share/doc/openvswitch/README.alpine
```

Reply:
It’d be better to just remove README.Debian and add README.alpine to the abuild as a file, not patch.

Reply:
fixed & rebased

Reply:
I’m very sorry for the mess, I’ve overlooked that this README file is much longer and only small part is changed to fit Alpine. 😿  Could you please revert this change and keep it as a patch? You can you `git reflog` to find SHA of the previous commit and restore it.

Reply:
reverting the sha reverted the whole PR